x-pack/plugins/security/public/account_management/index.ts

```diff
@@ -7,3 +7,4 @@
 
 export { accountManagementApp } from './account_management_app';
 export { UserProfileAPIClient } from './user_profile/user_profile_api_client';
+export type { UserProfileBulkGetParams, UserProfileGetCurrentParams } from './user_profile';
```

Comment:
__note:__ we might need to move `user_profile` to a top-level folder at some point, but I didn't want to have that many changes in this PR.